decryptor/postgresql/pg_hex_decryptor.go

```diff
@@ -142,12 +142,14 @@ func (decryptor *PgHexDecryptor) ReadSymmetricKey(privateKey *keys.PrivateKey, r
 	}
 	pubkey := &keys.PublicKey{Value: decryptor.decodedKeyBlockBuffer[:base.PublicKeyLength]}
 
-	smessage := message.New(privateKey, pubkey)
-	symmetricKey, err := smessage.Unwrap(decryptor.decodedKeyBlockBuffer[base.PublicKeyLength:])
-	if err != nil {
-		return nil, decryptor.keyBlockBuffer[:n], base.ErrFakeAcraStruct
+	for _, privateKey := range privateKeys {
```

Comment:
hmm, I thought I had removed hex/escape decoders.... will see on this parts again at some time. maybe forgot to delete old code...